fetching/service-worker.js

```diff
@@ -3,7 +3,7 @@
 // Create a proxy for all requests to the local urls containing a
 // `cookbook-proxy` string.
 self.onfetch = function(event) {
-  if (event.request.url.contains('cookbook-proxy')) {
+  if (event.request.url.match('cookbook-proxy')) {
```

Comment:
[contains][1] has been removed in Firefox 48 and replaced by [includes][2]. So you could use `includes` instead of `match`.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#String.prototype.contains
[2]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes